src/Apps/Order/Components/TermsOfServiceCheckbox.tsx

```diff
@@ -0,0 +1,22 @@
+import { color, Serif } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+import { Checkbox } from "Styleguide/Elements/Checkbox"
+
+export const TermsOfServiceCheckbox = props => {
+  return (
+    <Checkbox {...props}>
+      <Serif size="3" color={color("black60")}>
```

Comment:
I think you don't need the inner `color` wrapping here. It can just be `color="black60"`.